algorithms-miscellaneous-5/src/main/java/com/baeldung/algorithms/subarray/maximum/BruteForceAlgorithm.java

```diff
@@ -0,0 +1,30 @@
+package com.baeldung.algorithms.subarray.maximum;
```

Comment:
Could we maybe change the package name to maximumsubarray - which betters reflects the article and only has one level of nesting.